File: Assignment_13/program1.c
/////////////////////////////////////////////////////////////////
//
// Name :          program1.c
// Discription :   Display pattern
// Input :         Integer
// Output :        Charecter
// Author :        Shivam Rajendra Kale
// Date :          22/05/2025
//
/////////////////////////////////////////////////////////////////

//Input : Row = 4   col = 4

//Output : A  B  C  D
//         A  B  C  D
//         A  B  C  D
//         A  B  C  D

#include<stdio.h>

void pattern(int iROw, int iCol)
{
   int i = 0, j = 0;
   char ch = '\0';

   for(i = 1; i <= iROw; i++)
   {
        for(j = 1, ch = 'A'; j <= iCol; j++, ch++)
        {
           printf("%c\t",ch);
        }
        printf("\n");
   }
}

int main()
{
    int iValue1 = 0,iValue2 = 0;

    printf("Enter number of Row :");
    scanf("%d",&iValue1);

    printf("Enter number of Column :");
    scanf("%d",&iValue2);

    pattern(iValue1, iValue2);   //function call

    return 0;
}

File: Assignment_48/program1.java
//////////////////////////////////////////////////////////////////////////
//
// Name :          program1.java
// Discription :   use to Display pattern
// Input :         Integer
// Output :        Charecters
// Author :        Shivam Rajendra Kale
// Date :          13/10/2025
//
//////////////////////////////////////////////////////////////////////////

import java.util.*;

class pattern
{
    public void Display(int iRow, String str)
    {
        int i = 0, j = 0;
        int Arr[][] = new int [iRow][str.length()];
        char Brr[] = str.toCharArray();
       
        for(i = 0; i < iRow; i++)
        {
            for(j = 0; j < str.length(); j++)
            {
               System.out.print(Brr[j]+"\t");
            }
            System.out.println();
        }
    }
}

class program1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of rows :");
        int iRow = sobj.nextInt();

        sobj.nextLine();

        System.out.println("Enter a name :");
        String str = sobj.nextLine();

        pattern pobj = new pattern();

        pobj.Display(iRow,str);

        sobj.close();
    }
}